### PR TITLE
Upgrade sentry SDKs

### PR DESCRIPTION
**Pull request by rstorey**

### CODE REVIEW

File: concordia/celery.py [hunk 1]
```diff
@@ -2,16 +2,22 @@
...
 if SENTRY_BACKEND_DSN:
-    client = Client(SENTRY_BACKEND_DSN)
-    register_logger_signal(client)
-    register_signal(client)
+    CONCORDIA_ENVIRONMENT = os.environ.get("CONCORDIA_ENVIRONMENT", None)
+    sentry_sdk.init(
+        SENTRY_BACKEND_DSN,
+        environment=CONCORDIA_ENVIRONMENT,
+        release=get_concordia_version(),
+        integrations=[CeleryIntegration()],
```
Comment (acdha) [inline concordia/celery.py]: I'm wondering whether this should also include `DjangoIntegration()` — not sure whether that has any value if you're not using it in the traditional request/response cycle

File: concordia/templates/base.html [hunk 2]
```diff
@@ -171,17 +171,13 @@ <h2>Follow Us</h2>
...
-            if (navigator.userAgent.indexOf("CloudFlare-AlwaysOnline") < 0) {
-                Raven.config("{{ SENTRY_FRONTEND_DSN }}", {
-                    environment: "{{ CONCORDIA_ENVIRONMENT }}",
-                    release: "{{ RAVEN_CONFIG.release }}",
-                    ignoreUrls: [
-                        /^moz-extension/
-                    ]
-                }).install();
-            }
+            Sentry.init({
```
Comment (acdha) [inline concordia/templates/base.html]: I _think_ we're good with the filters for known crawlers but we should confirm that this doesn't spam us with the security restrictions for the CloudFlare Always Online crawler.

Comment (rstorey) [reply concordia/templates/base.html]: I added the condition back in - I didn't see anything in the docs about filters for known crawlers. Do you have any reference about it I could look into?

File: concordia/celery.py (hunk 1 above)
Comment (rstorey) [reply concordia/celery.py]: I don't think we need it, since there's no HTTP context.

**Pull request merged**